test/external/mlperf_bert/external_test_checkpoint_loading.py

```diff
@@ -0,0 +1,82 @@
+# Test whether pretrained weights from first BERT pretraining phase have been loaded correctly
+# Usage:
+# 1. Download the BERT checkoints with `wikipedia_download.py`
+#    Command: BASEDIR=/path/to/wiki python3 wikipedia_download.py
+# 2. Run this script. (Adjust EVAL_BS and GPUS as needed)
+#    Command: EVAL_BEAM=4 DEFAULT_FLOAT=half GPUS=6 BASEDIR=/path/to/wiki python3 test/external/mlperf_bert/external_test_checkpoint_loading.py
+import os
+from tqdm import tqdm
+
+from tinygrad.tensor import Tensor
+from tinygrad.device import Device
+from tinygrad.engine.jit import TinyJit
+from tinygrad.helpers import getenv
+from tinygrad.nn.state import get_state_dict
+from examples.mlperf.helpers import get_mlperf_bert_model, init_bert_from_checkpoint
+from examples.mlperf.dataloader import batch_load_val_bert
+
+# Copied from `examples/mlperf/model_train.py` in `train_bert``
```

Comment:
don't copy code like this. it should either reuse or be refactored out

Reply:
Ok, but I think we do not want to refactor out the JITs, right? - So, drop the tests?

Reply:
you can refactor a version that does not always use jit, and only apply jit wrapper when needed

Reply:
Sorry, I meant the eval_step and train_step functions. I think they are fine where they are.

Reply:
my point is that if you want to add test, don't copy code like this. it will get out of sync quickly

think about the best way to make sure the part you want to test is good after 6 months, even if you are no longer working actively on it